packages/react-native/ReactCommon/react/renderer/graphics/Transform.cpp

```diff
@@ -196,8 +197,14 @@ Transform Transform::FromTransformOperation(
         transformOperation.y.resolve(0),
         transformOperation.z.resolve(0));
   }
+  if (transformOperation.type == TransformOperationType::Arbitrary) {
+    auto arbitraryTransform = Transform{};
+    arbitraryTransform.operations.push_back(transformOperation);
+    arbitraryTransform.matrix = transform.matrix;
+    return arbitraryTransform;
+  }
 
-  // Identity or Arbitrary
+  // Identity
```

Comment:
Maybe instead, update the comment here to say "when using arbitrary transform, the caller is responsible for applying the value"